manifests/plugin.pp

```diff
@@ -27,10 +29,20 @@
       }
     }
 
-    concat::fragment { $title:
-      target  => $path,
-      content =>  template('hindsight/plugin.cfg.erb'),
-      order   => $order,
+    # if one gives a CONTENT or SOURCE arg, take that
+    if $content != undef or $source != undef {
+      concat::fragment { $title:
+        target  => $path,
+        content => $content,
+        source  => $source,
+        order   => $order;
```

Comment:
Normally we write comma at the end:
```puppet
order   => $order,
```

Reply:
Right. Today I read (the style guide)[https://puppet.com/docs/puppet/5.3/style_guide.html#resource-arrangement] and you're totally right. Will fix